src/frontend/src/hooks/usePIALookup.ts

```diff
@@ -5,21 +5,28 @@ import { IPIAIntake } from '../types/interfaces/pia-intake.interface';
 import { IPIAResults } from '../types/interfaces/pia-result.interface';
 import { HttpRequest } from '../utils/http-request.util';
 
-export const usePIALookup = () => {
+export const usePIALookup = (sortBy: string, sortOrder: number) => {
   const [tableData, setTableData] = useState<IPIAIntake[]>([]);
   useEffect(() => {
     (async () => {
+      let requestBody = {};
+      if (sortBy && sortOrder) {
+        requestBody = {
+          sortBy: sortBy,
```

Comment:
here maybe we need to use the query instead send the request body to the backend. you can check @kushal-arora-fw PR